src/freenet/clients/http/FProxyToadlet.java

```diff
@@ -399,7 +399,7 @@ public static String l10n(String msg) {
 	 * This is a horrible evil hack; we shouldn't be doing blacklisting, we should be doing whitelisting.
 	 * REDFLAG Expect future security issues!
 	 * @throws IOException */
-	private static boolean horribleEvilHack(Bucket data) throws IOException {
+	private static boolean isSniffedAsFeed(Bucket data) throws IOException {
```

Comment:
Oh, and can you move this to its own file? FproxyToadlet doesn’t really need to grow… :)